org-roam-macs.el

```diff
@@ -64,6 +64,21 @@ to look.
   (when org-roam-verbose
     (apply #'message `(,(concat "(org-roam) " format-string) ,@args))))
 
+(defmacro org-roam--get-completions (query &rest processing)
```

Comment:
can this be a function?

also, i'm a bit worried about bugs surfacing from the `completions` implicitly defined and returned here

Reply:
You're right.  This feels incredibly brittle.